client/templates/Macros/macros.html.twig

```diff
@@ -136,12 +136,20 @@
     </div>
 {% endmacro %}
 
-{% macro addAnotherSaveButton(button) %}
+{% macro addAnotherSaveButton(button, labelText, buttonClasses) %}
```

Comment:
I didn't know macro arguments are optional - saves updating each call to the macro!
You can also set default values for each argument apparently so:

`macro addAnotherSaveButton(button, labelText = 'continue', buttonClasses = 'govuk-button')`

Think your implementation works just as well though.